Anime-App/Feature/Anime/View/ViewControllers/AnimeViewController.swift

```diff
@@ -102,12 +114,35 @@ extension AnimeViewController: UISearchBarDelegate {
 
 extension AnimeViewController {
     
+    func addHapticFeedBack() {
+        let hapticFeedbackIsOn = UserDefaults.standard.bool(forKey: "UseHaptic")
+        if hapticFeedbackIsOn {
+            print("haptic feedback is on")
+            let generator = UIImpactFeedbackGenerator(style: .heavy)
+            generator.impactOccurred()
+        }
+    }
+    
+    @objc func addSound() {
+        let audioIsOn = UserDefaults.standard.bool(forKey: "UseAudio")
+        addHapticFeedBack()
```

Comment:
This shouldn't be in the function the only thing this should handle is playing sounds, this should be called outside of this function before or after you call the add sounds function you don't want to mix and match them